js/forum/dist/extension.js

```diff
@@ -62,6 +63,27 @@ System.register("wuethrich44/sso/main", ["flarum/extend", "flarum/app", "flarum/
                     items.remove('changeEmail');
                     items.remove('changePassword');
                 }
+
+                function checkRemoveAccountSection(items) {
+                    if (items.has('account')) {
```

Comment:
I've tested the code above and it works fine in my case. I just don't want to add 20 lines of code only to remove a simple heading. Do you like to simplify this or should we wait until https://github.com/flarum/core/pull/1218 is merged?